app/controllers/api/v1/notes_controller.rb

```diff
@@ -0,0 +1,36 @@
+module Api
+  module V1
+    class NotesController < ApplicationController
+      def index
+        render json: notes_filtered, status: :ok, each_serializer: IndexNoteSerializer
+      end
+
+      def show
+        render json: show_note, status: :ok, serializer: ShowNoteSerializer
```

Comment:
y a esto con el singular
```suggestion
        render json: note, status: :ok, serializer: ShowNoteSerializer
```